server/health/index.js

```diff
@@ -1 +1,68 @@
-export * from "./healthchecks";
+import { database as db } from "../models";
+
+export function runHealthChecks() {
+    // Checks database connectivity by auth attempt
+    function databaseConnectivityCheck() {
+        return db.authenticate();
+    }
+
+    function handleResolve({ label, promise }) {
+        return { label, promise: promise.then(() => ({ [label]: true })) };
```

Comment:
This line is confusing, why we need the first label? In `handleReject` we don't.

Reply:
We pass the label to the function to be able to output a name with the health check. `handleResolve` is mapped over first: We have to return label again from there to be able to use it in the following `handleReject`. We don't return the label from there because we no longer need it (resolve and reject use it already) and more importantly: we are required to procue a list of promises, not obejcts with a label and a promise.